homeassistant/components/netatmo/data_handler.py

```diff
@@ -69,7 +70,9 @@ async def async_setup(self):
         )
 
         self.listeners.append(
-            self.hass.bus.async_listen("netatmo_event", self.handle_event)
+            async_dispatcher_connect(
+                self.hass, f"signal-{DOMAIN}-webhook-None", self.handle_event,
```

Comment:
When is the event type `None`? Is that what we're connecting to here?

Reply:
Yes, there are "system" related events like the acknowledgement of the established webhook that don't return an event type.